### ISSUE TITLE
SSD super slow AtlasOS 20H2

### ISSUE THREAD

**Issue by maverikh** [Atlas-OS/Atlas#448]
### Prerequisites

- [X] This is not a support issue or a question. For any support, questions or help, visit our [Discord server](https://discord.com/servers/atlas-795710270000332800).
- [X] I performed a [cursory search of the issue tracker](https://github.com/Atlas-OS/Atlas/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I checked the [documentation](https://github.com/Atlas-OS/Atlas/wiki) to understand that the issue I am reporting is not normal behavior.
- [X] I understand that not filling out this template will lead to the issue being closed.

### Description

Since installed AtlasOS 20H2 my onboard Intel RST Raid-5 is super slow... its made of 3 Samsung 980PRO NVME's.

### Steps to reproduce (if applicable add screenshots)

Run CrystalDiskMark 8.0.4
SSD is running at 1400.00 MB/s or less.

### Expected behavior

SSD Raid5 on normal windows install 13,500.00+ MB/s

### Actual behavior (if applicable add screenshots)

Currently only running at 400.00 MB/s

### Atlas Version (1803/20H2/21H2)

20H2

### Desktop information

Intel i9-12900K @ 5.2GHz
G.Skill 32GB @ 6400
3x Samsung 980 PRO SSD's in RAID-5
Asus TUF 3090Ti

**Comment by isipisi89** [comment]
try the samsung nvme driver https://semiconductor.samsung.com/consumer-storage/support/tools/

**Comment by maverikh** [comment]
The actual driver download doesn't suppot the 980 PRO and says so when you try to install it. I tried Samsung magician and it have latest firmware. No resulting change.

**Comment by Xyueta** [comment]
> Try installing this [update](https://support.microsoft.com/en-us/topic/november-22-2021-kb5007262-os-build-22000-348-preview-7f3e18d7-4189-4882-b0e9-afc920253aee). since atlas disabled windows update, you can get it from the microsoft update catalog. Here is a [fast link](https://www.catalog.update.microsoft.com/Search.aspx?q=KB5007262)

It is for Windows 11.

**Comment by PencilNavigator** [comment]
> > Try installing this [update](https://support.microsoft.com/en-us/topic/november-22-2021-kb5007262-os-build-22000-348-preview-7f3e18d7-4189-4882-b0e9-afc920253aee). since atlas disabled windows update, you can get it from the microsoft update catalog. Here is a [fast link](https://www.catalog.update.microsoft.com/Search.aspx?q=KB5007262)
> 
> It is for Windows 11.

whoops, sorry for that.

**Comment by maverikh** [comment]
So tried windows 10 from standard install media and issue went away. I
build an nlite is for my own use and includes the nvme driver from above,
this caused the install process to fail when loading that driver.

I installed windows 11 from Ms website and the problem goes away.

**Comment by maverikh** [comment]
Did a full re-intall os 20H2... Same issue... Perhaps 22H2 has fix??

**Comment by maverikh** [comment]
So I reinstalled atlasos 20h2 on a virtual machine and when the installer ran the data rate wasnear 10k.00 but once os booted... It slowed back down.

I talked with a coworker that does data storage architecture for a living and he thought it is an issue with the way atlasos is reading the onboard controller bus capability. I put in some Samsung 970 PRO nvme drives from my old computer and I installed atlasos... You guessed it my data speed reached the 8k.00 range... I don't think the 970's are as fast but they do support the Samsung driver where the 989Pro does not.
